ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disconnected/fullyprivate/cco-manual-workload-identity/provision/OWNERS

```diff
@@ -0,0 +1,16 @@
+approvers:
+- jianlinliu
+- Amoghrd
```

Comment:
Here you can remove Amoghrd mgahagan73, and replace MayXuQQ with @jinyunma

Reply:
Sorry, missed it